index.html

```diff
@@ -0,0 +1,213 @@
+                <li class="item-m"><a href="#interviwes"  class="btn-m">Interviwes</a></li>
+                <li class="item-m"><a href="#home  " class="btn-m">Sponsor</a></li>
+                <li class="item-m"><a href="#home" class="btn-m">News</a></li>
+                <li class="item-m out"><a href="#home" class="btn-m">CC Tickets</a></li>
+            </ul>
+            <div id="menu-account">
+                <a class="menu-link" href="#home">Mypage</a>
+                <a class="menu-link" href="#home">Logout</a>
+            </div>
+        </div>
+    </div>
+    <section id="program">
+        <h2 id="program-h">Line-Up</h2>
+        <hr>
+        <div id="program-grid-container">
+            <ul id="program-grid" >
+               <li class="grid-item">
+                <img class="program-img" alt="image-band"  src="css/img/band-1.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">RHCP</h3>
+                </div>
+                <p class="program-p">RED HOME CHILLI PEPPERS in Live on Sunday. Playing while 4 hours!</p>
+               </li>
+               <li class="grid-item">
+                <img class="program-img" alt="image-band" src="css/img/band-2.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">FooFingers</h3>
+                </div>
+                <p class="program-p">FOO FINGERS in Live on Friday. Playing while 2 hours!</p>
+               </li>
+               <li class="grid-item">
+                <img class="program-img" alt="image-band" src="css/img/band-3.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">BonJovi</h3>
+                </div>
+                <p class="program-p">BON JOVI in Live on Friday. Playing while 3 hours!</p>
+               </li>
+               <li class="grid-item">
+                <img class="program-img" alt="image-band" src="css/img/band-4.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">4NOS</h3>
+                </div>
+                <p class="program-p">CUARTETO DE NOS in Live on Saturday. Playing while 3 hours!</p>
+               </li>
+               <li class="grid-item">
+                <img class="program-img" alt="image-band" src="css/img/band-5.jpg">
+                <div class="cnt-title">                
+                    <h3 class="program-title">Milky</h3>
+                </div>
+                <p class="program-p">MILKY CHANCE in Live on Saturday. Playing while 2 hours!</p>
+               </li>
+            </ul>
+        </div>
+        <button id="join">Yesterland Tickets</button>
+    </section>
+    <section id="about" class="about">
+        <div id="about-container">
+            <h3 class="small-heading">Arrives in Urugay...</h3>
+            <h2 id="big-heading-about">   
```

Comment:
- [ ] I suggest that you match the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided.  For this reason, I suggest that the `about` is added to its own page and all the sections of the `about menu` should be added. 
- [ ] In addition kindly follow the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) when designing this about page ☺️.

Original design 😎 
![image](https://github.com/BiancaMe/Capstone/assets/99394374/603e2f7d-8ec9-44ff-a19c-cb23982867c6)